include/Core/Config.php

```diff
@@ -132,15 +132,33 @@ public static function set($family,$key,$value) {
 		$dbvalue = (is_array($value)?serialize($value):$value);
 		$dbvalue = (is_bool($dbvalue) ? intval($dbvalue) : $dbvalue);
 
-		$ret = q("INSERT INTO `config` ( `cat`, `k`, `v` ) VALUES ( '%s', '%s', '%s' )
-ON DUPLICATE KEY UPDATE `v` = '%s'",
+		// The "INSERT" command is very cost intense. It saves performance to do it this way.
```

Comment:
Including the get (instead of the first sql query) is a good idea. I will adopt it. But concerning the other part I think we need the separation of "insert" and "update" for performance reasons.